doc/source/tune-searchalg.rst

```diff
@@ -0,0 +1,81 @@
+Tune Search Algorithms
+======================
+
+Tune provides various hyperparameter search algorithms to efficiently optimize your model. Tune allows you to use different search algorithms in combination with different trial schedulers. Tune will by default implicitly use the Variant Generation algorithm to create trials.
+
+.. important:: If you **explicitly** specify a Search Algorithm, you will need to pass in the experiment configuration into the Search Algorithm initializer.
```

Comment:
Move this to after the example block below.

Reword: note that the `search_alg` arg replaces passing in the experiment configuration directly.

Reply:
This is also just kind of weird API-wise, can we do better here?

Reply:
Yeah I agree; perhaps something where you can pass a str identifier for the search algorithm, put the search space in `config`, and handle the linking together underneath?

i.e.,
```
{"run": f,
 "config": {hyperopt.space},
 "searchalg": "HyperOpt" }